pkg/autoscaler/metrics/stats_scraper_test.go

```diff
@@ -205,19 +201,25 @@ func TestPodDirectScrapeSuccess(t *testing.T) {
 }
 
 func TestPodDirectScrapeSomeFailButSuccess(t *testing.T) {
-	fake.KubeClient = fakek8s.NewSimpleClientset()
-	fake.KubeInformer = kubeinformers.NewSharedInformerFactory(fake.KubeClient, 0)
-
 	// For 5 pods, we need 4 successes.
```

Comment:
Comment seems not in the right place.

Reply:
I think it's ok. It describes the test expectations.